src/main.rs

```diff
@@ -111,6 +111,13 @@ fn create_app<'a, 'b>(
                 .takes_value(true)
```

Comment:
We like to prefix our commit messages with the area of the code. So this might be better as "main, vmm: add --initramfs option"

Reply:
ok